obspy/core/frequency_domain_trace.py

```diff
@@ -0,0 +1,245 @@
+from obspy.core.trace import BaseTrace, Stats
+from matplotlib import mlab
+from numpy import fft
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy import fftpack, signal, stats
+from obspy.core.trace import Trace
+
+class TimeSeriesFrequencyDomainTrace(BaseTrace):
+
+    def ifft(self):
+        from obspy.core.trace import TimeSeriesTrace
+        t_tr = TimeSeriesTrace()
+        t_tr.data = fft.irfft(self.data)
+        t_tr.stats = Stats()
+        t_tr.stats = self.stats
+        return t_tr
+
+    def plot(self):
+        N = self.stats.npts
+        T = self.stats.delta
+        yf = self.data
+        xf = np.linspace(0.0, 1.0/(2.0*T), N/2)
+        fig, ax = plt.subplots()
+        ax.plot(xf, 2.0/N * np.abs(yf[:N/2]))
```

Comment:
This should just use the `self.frequencies()` method which would me much easier.